packages/core/src/components/menu/menuItem.tsx

```diff
@@ -26,6 +26,13 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
      */
     label?: string | JSX.Element;
 
+    /**
+     * Whether the text should be allowed to wrap to multiple lines.
+     * If `false`, text will be truncated with an ellipsis when it reaches `max-width`.
+     * @default false
+     */
+    multiline?: boolean;
```

Comment:
🎩

Reply:
@llorca thoughts on the centered icon alignment? (see your screenshot in comment below)

Reply:
good call, actually aligned at the top would be better in this case